dash_bio/component_factory/_manhattan.py

```diff
@@ -78,80 +78,76 @@ def ManhattanPlot(
 
 Keyword arguments:
 - dataframe (dataframe; required): A pandas dataframe which must contain at
-    least the following  three columns:
+    least the following three columns:
             - the chromosome number
             - genomic base-pair position
             - a numeric quantity to plot such as a p-value or zscore
-- chrm (string; optional): A string denoting the column name for the
-    chromosome.  This column must be float or integer.  Minimum number
-    of chromosomes required is 1. If you have X, Y, or MT chromosomes,
-    be sure to renumber these 23, 24, 25, etc. (Default: "CHR")
-- bp (string; optional): A string denoting the column name for the
-    chromosomal position. (Default: "BP")
-- p (string; optional): A string denoting the column name for the
+- chrm (string; default 'CHR'): A string denoting the column name for
+    the chromosome.  This column must be float or integer.  Minimum
```

Comment:
```suggestion
    the chromosome. This column must be float or integer. Minimum
```
English style conventions are different in Python (docstrings, comments).